RELEASING.md

```diff
@@ -0,0 +1,20 @@
+# Releasing
+
+Releases of comit-rs are mostly automated.
+Here is what you need to know:
+
+The repository uses the GitFlow branching model.
+Hence, releases are started by branching a release-branch (`release/x.y.z`) of `dev`.
+
+1. Start with creating said release branch locally.
+2. Update the changelog to reflect the newest release.
+3. Bump necessary versions in the manifest files.
+4. Commit and push your changes
+5. Create a pull request targeting the __master__ branch.
+6. Get approvals and merge it.
+
+From here, [the automation](./.github/workflows/publish-new-release.yml) takes over and:
+
+1. Build the release artifacts for Linux and MacOS.
+2. Pushes a new tag of the [comitnetwork/cnd](https://hub.docker.com/repository/docker/comitnetwork/cnd) docker image.
+3. Tags the merge commit and creates a release on GitHub with the binaries attached.
```

Comment:
```suggestion
3. Tags the merge commit and creates a release on GitHub with the binaries attached.
4. Opens a PR to merge back to _dev_ branch, do ensure it gets merged.
```
Or do the job merge the PR automatically?

Reply:
It just opens it, good catch!